### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Manvendra2000**
Consolidated the pointers into a single line and improved the documentation's readability.

**Notes for Reviewers**

This PR fixes #
Updated the subsequent screenshot

Signing off as Manvendra [mzealpro@gmail.com]

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✅ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for this, @Manvendra2000. Will you please instead remove this image and copy/paste the text, checkmarks, and community icon from https://github.com/layer5io/layer5/blob/master/README.md ?

**Comment by Manvendra2000** [comment]
> Thanks for this, @Manvendra2000. Will you please instead remove this image and copy/paste the text, checkmarks, and community icon from https://github.com/layer5io/layer5/blob/master/README.md ?

For Sure! [here ](https://github.com/layer5io/docs/pull/391/commits/6d095130d7bbb5e2801fab280f0426f44ca81058)it is @leecalcote

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
-![image](https://github.com/user-attachments/assets/0d802112-175c-437b-947f-a63b8b483436) # Layer5 Product Documentation
-
+<div class="flex">
```
Comment (leecalcote) [inline README.md:1]: is flex used on the original readme? I'd be surprised if GFMD supported this.

**Review by leecalcote — COMMENTED** [review]
Thanks!

**Comment by leecalcote** [comment]
@Manvendra2000 https://layer5io.slack.com/archives/C019426UBNY/p1729177173523569

File: README.md (hunk 1 above)
Comment (Manvendra2000) [reply README.md]: Hey @leecalcote I have removed class="flex" in the new commit. Apart from that the in the doc in meshary/readme.md line 313 below is the screenshot, flex is being used. should I work on this issue image

**Comment by vishalvivekm** [comment]
@Manvendra2000 
Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by Manvendra2000** [comment]
@vishalvivekm I have added the the issue to Agenda of meeting.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
-![image](https://github.com/user-attachments/assets/926e71c4-4f1b-4188-8540-6f951c173710)# Layer5 Product Documentation
-
+<div>
+<a href="https://meshery.io/community"><img alt="Layer5 Community" src=".github/readme/images//community.svg" style="margin-right:8px;padding-top:9px;" width="140px" align="left" /></a>
+<p>
+✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
+✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
```
Comment (vishalvivekm) [inline README.md:5]:
```suggestion
✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/Layer5io?sub_confirmation=1">meeting recordings</a>.<br />
```

**Pull request merged**